rmc/utils/simultaneous_breaks.py

```diff
@@ -383,7 +385,12 @@ def search_for_two_breaks(
     )
     if save_chisq_ht:
         group_ht = group_ht.checkpoint(
-            f"{SIMUL_BREAK_TEMP_PATH}/temp_chisq_over_threshold.ht", overwrite=True
+            f"{SIMUL_BREAK_TEMP_PATH}/dataproc_temp_chisq_group{count}.ht",
+            overwrite=True,
+        )
+    else:
+        group_ht = group_ht.checkpoint(
+            f"{TEMP_PATH_WITH_DEL}/dataproc_temp_chisq.ht", overwrite=True
```

Comment:
If we are running multiple jobs with save_chisq_ht=False, this could break since they could overwrite each other while running, right? In the batch script, this still has the counter on it

Reply:
that's true! I only typically spin up one cluster at a time to run the dataproc jobs, though, so this isn't currently an issue